### DISCUSSION TITLE
Support version 1.7.0 of CIS Benchmarks for Kubernetes 1.25.X

### DISCUSSION THREAD

**Discussion by Jamison1** [aquasecurity/kube-bench#1420] [Development]
The CIS benchmark for Kubernetes v1.25 is out as of 3/20/2023

The latest Kubernetes version currently supported in kube-bench is v1.24

Unlike the prior versioning scheme, this one goes back to the old scheme which covered several Kubernetes versions.

Example: CIS Benchmark v1.6.1 covered Kubernetes v1.16 - v1.18

[Example of versions matching rancher/rke2 versioning:](https://github.com/rancher/rke2/issues/4103)
CIS Benchmark v1.6.1 for K8s v1.18 Last updated 10/01/2020
CIS Benchmark v1.23 for K8s v1.23 Last updated 5/13/2022
CIS Benchmark v1.24 for K8s v1.24 Last updated 9/21/2022
CIS Benchmark v1.7.0 for K8s v1.25 Last updated 3/20/2023

**Comment by mozillazg** [comment]
Thanks for your reminder! I'll take a look at it when I have free time.
